src/PCIVendor.php

```diff
@@ -91,19 +94,27 @@ public function rawSearchOptions()
      */
     public static function getList(): array
     {
-        /** @var \Psr\SimpleCache\CacheInterface $GLPI_CACHE */
+        /** @var CacheInterface $GLPI_CACHE */
         global $GLPI_CACHE;
 
         $vendors = new PCIVendor();
-        if (($pciids = $GLPI_CACHE->get($vendors->cache_key)) !== null) {
-            return $pciids;
+        try {
+            if (($pciids = $GLPI_CACHE->get($vendors->cache_key)) !== null) {
+                return $pciids;
+            }
+        } catch (Psr\SimpleCache\InvalidArgumentException $e) {
```

Comment:
Is this something we do now ? I was not aware of changes in recommendation with cache management.
Maybe I missed it.